pkg/config/config.go

```diff
@@ -23,7 +23,11 @@ type Feed struct {
 	// Format is "300ms", "1.5h" or "2h45m".
 	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 	// NOTE: too often update check might drain your API token.
+	// todo deprecated, remove that someday
 	UpdatePeriod Duration `toml:"update_period"`
+	// Cron expression format is how often to check update
+	// NOTE: too often update check might drain your API token.
+	Schedule string `toml:"schedule"`
```

Comment:
Can we name this `CronSchedule string toml:"cron_schedule"` to underline that cron expressions expected here.